docs/standard/library-guidance/sourcelink.md

```diff
@@ -32,6 +32,10 @@ You can use [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuG
 
 > For the best debugging experience your library should publish symbol files as well as use Source Link. For more information about symbol files and symbol packages, see [Symbol packages](./nuget.md#symbol-packages).
 
+✔️ CONSIDER enabling deterministic builds.
+
+> Deterministic builds enable verification that the resulting binary was built from the specified source and provides traceability. For more information about deterministic builds, and instructions for enabling them, see [Deterministic Builds](https://github.com/clairernovotny/DeterministicBuilds).
```

Comment:
```suggestion
> Deterministic builds enable verification that the resulting binary was built from the specified source and provide traceability. For more information about deterministic builds and instructions for enabling them, see [Deterministic Builds](https://github.com/clairernovotny/DeterministicBuilds).
```

Reply:
> Deterministic builds enable verification that the resulting binary was built from the specified source and provides traceability.

What tool is providing this verification?

Reply:
We don't have a tool yet to do the verification. Getting the Source Link and deterministic settings are a prerequisite for any tool to work.